modules/ROOT/pages/setup.adoc

```diff
@@ -66,7 +66,7 @@ The wrapper.conf file is located at `$MULE_HOME/conf/wrapper.conf`.
 From that location you can locate the `wrapper.conf` user conf directory.
 
 IMPORTANT: These are example values. If you are unsure about the Mule runtime capacity you need to handle your B2B transactions,
-please contact your MuleSoft account representative for assistance
+please contact your MuleSoft account representative for assistance.
```

Comment:
OMIT, per above, or move to follow the example.

Reply:
Next release.